src/script/components/MessagesList/Message/MessageWrapper.tsx

```diff
@@ -78,7 +78,7 @@ export const MessageWrapper: React.FC<MessageParams & {hasMarker: boolean; isMes
   isMsgElementsFocusable,
 }) => {
   const findMessage = (conversation: Conversation, messageId: string) => {
-    return messageRepository.getMessageInConversationById(conversation, messageId, true, true);
+    return messageRepository.getMessageInConversationByReplacementId(conversation, messageId);
```

Comment:
Humm I think we also need to get the message by regular ID at this point. The replacing_id should only be a fallback in case we don't find the message by ID

Reply:
i think the naming is confusing you here but it does indeed to either/or, not only replacingId